util/otbn_build.py

```diff
@@ -127,6 +130,43 @@ def call_rv32_ar(args: List[str]):
     run_cmd([rv32_tool_ar] + args)
 
 
+def get_otbn_syms(elf_path: str) -> List[Tuple[str, int]]:
+    '''Get externally-visible symbols from an ELF
+
+    Symbols are returned as a list of triples: (name, address). This
+    discards locals and also anything in .scratchpad, since those addresses
+    aren't bus-accessible.
+    '''
+    with tempfile.TemporaryDirectory() as tmpdir:
+        # First, run objcopy to discard local symbols and the .scratchpad
+        # section. We also use --extract-symbol since we don't care about
+        # anything but the symbol data anyway.
+        syms_path = os.path.join(tmpdir, 'syms.elf')
+        call_rv32_objcopy(['-O', 'elf32-littleriscv',
```

Comment:
What does this objcopy accomplish, really? Shouldn't the ELF reader library be able to do that?